dotcom-rendering/src/types/content.ts

```diff
@@ -1008,16 +1008,18 @@ export type KnowledgeQuizAtomType = {
 	id: string;
 	questions: QuestionType[];
 	resultGroups: ResultGroupsType[];
-	sharingUrls: SharingUrlsType;
-	theme: ArticleTheme;
+	pageId: string;
+	webTitle: string;
+	format: ArticleFormat;
 };
 
 export type PersonalityQuizAtomType = {
 	id: string;
 	questions: QuestionType[];
 	resultBuckets: ResultsBucketType[];
-	sharingUrls: SharingUrlsType;
-	theme: ArticleTheme;
+	pageId: string;
+	webTitle: string;
+	format: ArticleFormat;
 };
 
```

Comment:
The following types can be safely removed from this file: `SharePlatformType`, `SharingUrlsType`